angular_gallery/lib/builder/gallery_app_builder.dart

```diff
@@ -32,7 +32,7 @@ class GalleryWebBuilder extends Builder {
   Future _generateMainDart(BuildStep buildStep) async {
     final mustacheContext = {
       'galleryImportUri':
-          'package:${buildStep.inputId.package}/gallery/gallery.dart'
+          'package:${buildStep.inputId.package}/gallery/gallery.template.dart'
     };
     final newAssetId = new AssetId(buildStep.inputId.package, 'web/main.dart');
```

Comment:
Don't need `new` anymore.

Reply:
We haven't started cleaning these up internally yet. We were told to wait until some edge cases were solved internally. I'd rather not do them piecewise, but cleanup a large swatch when we get the all clear.